### DISCUSSION TITLE
Basics of Python

### DISCUSSION THREAD

**Discussion by nishumaxter** [TheAlgorithms/Python#8148] [General]
Hi everyone.

I am into healthcare and want to lean Python for Genomic data science screening. Can anyone help.

**Comment by nam3isrobin** [comment]
> Hi everyone.
> 
> I am into healthcare and want to lean Python for Genomic data science screening. Can anyone help. 

I can send you notes

**Reply by G3X2001** [reply]
Could you also send me the notes?

**Comment by Kmugoz** [comment]
Me too ✋️ notes plz
